src/duiker.py

```diff
@@ -380,7 +380,7 @@ def parse_args(argv):
         PS1="\\u@\\h:\\w$ "
     }}
 
-    PROMPT_COMMAND=__prompt
+    PROMPT_COMMAND='(__prompt $)'
```

Comment:
I did not mean for this to get wrapped up in the PR. I'm experimenting with putting the Python call in the background for less perceived delay in the terminal. That syntax prevents the process from echoing status on every call, but may not be the best approach.